### PR TITLE
Added Models With Test Cases

### PR DESCRIPTION
**Pull request by kashifkhan0771**
Added Models With Test Cases

### CODE REVIEW

File: models/host.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package models
+
+import "github.com/fatih/structs"
+
+// Host Model
+type Host struct {
+	ID       string                 `struct:"ID"`
+	IP       string                 `struct:"IP"`
+	Name     string                 `struct:"Name"`
+	MetaData map[string]interface{} `struct:"MetaData"`
```
Comment (ahab94) [inline models/host.go:10]: Metadata

File: models/host.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package models
+
+import "github.com/fatih/structs"
+
+// Host Model
+type Host struct {
+	ID       string                 `struct:"ID"`
+	IP       string                 `struct:"IP"`
+	Name     string                 `struct:"Name"`
```
Comment (ahab94) [inline models/host.go:9]: Tags should be snake case.

 Also struct should be structs

Also add json and bson tags

Comment (ahab94) [reply models/host.go:9]: https://github.com/ahab94/ApnaSchool/blob/dev/models/student.go

File: models/host.go [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package models
+
+import "github.com/fatih/structs"
+
+// Host Model
+type Host struct {
+	ID       string                 `structs:"id" json:"id" bson:"_id" db:"id"`
+	IP       string                 `structs:"ip" json:"ip" bson:"ip" db:"ip"`
+	Name     string                 `structs:"name" json:"name" bson:"name" db:"name"`
+	Metadata map[string]interface{} `structs:"metaData" json:"metaData" bson:"metaData" db:"metaData"`
```
Comment (ahab94) [inline models/host.go:10]: Tag should be metadata and not metaData

File: models/host_test.go [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+					"Os":      "Linux",
+					"Version": "17.2",
+				},
+				Status: true,
+			},
+			want: map[string]interface{}{
+				"id":   "Host 1",
+				"ip":   "192.168.122.236",
+				"name": "Linux",
+				"metaData": map[string]interface{}{
```
Comment (ahab94) [inline models/host_test.go:37]: Should br metadata

File: models/host_test.go [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+				ID:   "Host 1",
+				IP:   "192.168.122.236",
+				Name: "Linux",
+				Metadata: map[string]interface{}{
+					"Os":      "Linux",
+					"Version": "17.2",
+				},
+				Status: true,
+			},
+			want: []string{"ID", "IP", "Name", "Metadata", "Status"},
```
Comment (ahab94) [inline models/host_test.go:101]: Names shouldn't be capital... should always be lower snake case

**Review by ahab94 — CHANGES_REQUESTED** [review]
More feedback

File: models/host_test.go (hunk 5 above)
Comment (kashifkhan0771) [reply models/host_test.go:101]: Okay i will fix this ASAP

**Comment by kashifkhan0771** [comment]
Previous Issues are fixed. Please check again and let me know in case of issues or new changes.

**Pull request merged**